fairlearn/postprocessing/_interpolated_thresholder.py

```diff
@@ -129,3 +131,13 @@ def predict(self, X, *, sensitive_features, random_state=None):
         positive_probs = self._pmf_predict(
             X, sensitive_features=sensitive_features)[:, 1]
         return (positive_probs >= random_state.rand(len(positive_probs))) * 1
+
+    def _print_thresholds(self):
```

Comment:
what about a public `get_thresholds()` method which returns a `dict` and if we wanna print it nicely, we can use `pprint` or `print(json.dumps())`

Reply:
I like that for another reason: someone asked me recently if they could get the thresholds that it comes up with and change them as needed. For example, this could be used to simplify them a bit and then plug it right back in. Perhaps that's a bit more than you thought, but this would make it easier :-) Will work this in right away

Reply:
I just realized that `InterpolatedThresholder`, the class on which I wrote the `thresholds()` method, is currently private since it's only used internally. I'm sort of torn between making it public by adding it to the `__init__.py` file of the postprocessing module and moving the `thresholds()` method to `ThresholdOptimizer` since that's basically the main class people are interacting with. This will also need slight adjustments for the case we discussed in #660 if we go with the latter, but nothing dramatic.

Reply:
Went with the latter option for now since it strikes me as more intuitive for users.

Reply:
closing this thread since it doesn't exist anymore (see other thread)